Applications/DataExplorer/DataView/GEOModels.h

```diff
@@ -76,6 +76,9 @@ public slots:
     void appendSurfaceVec(std::string const& name);
     void removeSurfaceVec(std::string const& name);
 
+    void renameGeometry(std::string const& old_name,
```

Comment:
I'd rather like to see docu of the GeoObjects::renameGeometry interface (mentioned below)...